From d62476a74923f83055c184df7d63bab533ce6a0f Mon Sep 17 00:00:00 2001
From: Allen Wild <allenwild93@gmail.com>
Date: Sun, 13 Aug 2017 13:37:01 -0400
Subject: [PATCH 19/19] neostrip, shiftbrite: include uaccess.h from linux/
 rather than asm/

The definitions for copy_(to|from)_user moved
---
 drivers/spi/neostrip.c   | 2 +-
 drivers/spi/shiftbrite.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/spi/neostrip.c b/drivers/spi/neostrip.c
index 0f533ac948f0..02840c18d640 100644
--- a/drivers/spi/neostrip.c
+++ b/drivers/spi/neostrip.c
@@ -24,7 +24,7 @@
 #include <linux/spinlock.h>
 #include <linux/spi/spi.h>
 #include <linux/string.h>
-#include <asm/uaccess.h>
+#include <linux/uaccess.h>
 #include <uapi/spi/neostrip.h>
 
 // definitions
diff --git a/drivers/spi/shiftbrite.c b/drivers/spi/shiftbrite.c
index 1f7afaed5477..0410c113d9be 100644
--- a/drivers/spi/shiftbrite.c
+++ b/drivers/spi/shiftbrite.c
@@ -24,7 +24,7 @@
 #include <linux/of_gpio.h>
 #include <linux/spinlock.h>
 #include <linux/spi/spi.h>
-#include <asm/uaccess.h>
+#include <linux/uaccess.h>
 
 #include <uapi/spi/shiftbrite.h>
 
-- 
2.19.1

